pkg/drivers/kic/oci/network_create.go

```diff
@@ -36,6 +37,9 @@ const firstSubnetAddr = "192.168.49.0"
 // big enough for a cluster of 254 nodes
 const defaultSubnetMask = 24
 
+// name of the bridge network that docker creates by default to be used to get the MTU. ( related issue #9528)
```

Comment:
Alternative wording:

`// name of the default Docker bridge network, used to lookup the MTU (see #9528)`

At a minimum though, remove space after (